### ISSUE TITLE
dendron.topic.publish.cli: example instructions are confusing

### ISSUE THREAD

**Issue by l2dy** [dendronhq/dendron-site#325]
https://github.com/dendronhq/dendron-site/pull/310 makes the [Publish CLI](https://wiki.dendron.so/notes/yQVhJtdQ40n3SLHJKAeeU/) page confusing as if an npx one-liner would work without context in the referenced page.

`npx` would not know that `dendron` is from @dendronhq/dendron-cli if it's not installed.

**Comment by hrmnjt** [comment]
Agreed. Suggestion as mentioned on #310 - https://github.com/dendronhq/dendron-site/pull/310#issuecomment-985083462

Better would be to document behavior in both cases when a user does - `npx install @dendronhq/dendron-cli` or `npx install -g @dendronhq/dendron-cli`

I think #310 is causing inconsistencies on [publish > cook > github](https://wiki.dendron.so/notes/yg3EL1x9fEe4NMqxUC3jP/) and [cli](https://wiki.dendron.so/notes/23a1b942-99af-45c8-8116-4f4bb7dccd21/) pages as well.
